CHANGELOG.rst

```diff
@@ -382,37 +382,37 @@ Features & Improvements
+- Added support for python 3.7 via a few small compatibility / bug fixes.  `#2427 <https://github.com/pypa/pipenv/issues/2427>`_,
   `#2434 <https://github.com/pypa/pipenv/issues/2434>`_,
   `#2436 <https://github.com/pypa/pipenv/issues/2436>`_
 
 - Added new flag ``pipenv --support`` to replace the diagnostic command ``python -m pipenv.help``.  `#2477 <https://github.com/pypa/pipenv/issues/2477>`_,
   `#2478 <https://github.com/pypa/pipenv/issues/2478>`_
 
-- Improved import times and CLI runtimes with minor tweaks.  `#2485 <https://github.com/pypa/pipenv/issues/2485>`_
+- Improved import times and CLI run times with minor tweaks.  `#2485 <https://github.com/pypa/pipenv/issues/2485>`_
 
 
 Bug Fixes
 ---------
 
-- Fixed an ongoing bug which sometimes resolved incompatible versions into lockfiles.  `#1901 <https://github.com/pypa/pipenv/issues/1901>`_
+- Fixed an ongoing bug which sometimes resolved incompatible versions into the project Lockfile.  `#1901 <https://github.com/pypa/pipenv/issues/1901>`_
 
 - Fixed a bug which caused errors when creating virtualenvs which contained leading dash characters.  `#2415 <https://github.com/pypa/pipenv/issues/2415>`_
 
-- Fixed a logic error which caused ``--deploy --system`` to overwrite editable vcs packages in the pipfile before installing, which caused any installation to fail by default.  `#2417 <https://github.com/pypa/pipenv/issues/2417>`_
+- Fixed a logic error which caused ``--deploy --system`` to overwrite editable vcs packages in the Pipfile before installing, which caused any installation to fail by default.  `#2417 <https://github.com/pypa/pipenv/issues/2417>`_
 
 - Updated requirementslib to fix an issue with properly quoting markers in VCS requirements.  `#2419 <https://github.com/pypa/pipenv/issues/2419>`_
 
 - Installed new vendored jinja2 templates for ``click-completion`` which were causing template errors for users with completion enabled.  `#2422 <https://github.com/pypa/pipenv/issues/2422>`_
 
-- Added support for python 3.7 via a few small compatibility / bugfixes.  `#2427 <https://github.com/pypa/pipenv/issues/2427>`_
+- Added support for python 3.7 via a few small compatibility / bug fixes.  `#2427 <https://github.com/pypa/pipenv/issues/2427>`_
 
 - Fixed an issue reading package names from ``setup.py`` files in projects which imported utilities such as ``versioneer``.  `#2433 <https://github.com/pypa/pipenv/issues/2433>`_
 
 - Pipenv will now ensure that its internal package names registry files are written with unicode strings.  `#2450 <https://github.com/pypa/pipenv/issues/2450>`_
 
 - Fixed a bug causing requirements input as relative paths to be output as absolute paths or URIs.
-  Fixed a bug affecting normalization of ``git+git@host`` uris.  `#2453 <https://github.com/pypa/pipenv/issues/2453>`_
+  Fixed a bug affecting normalization of ``git+git@host`` urls.  `#2453 <https://github.com/pypa/pipenv/issues/2453>`_
```

Comment:
URI and URL are different things (ai’m not sure which is better here). But why lower case?

Reply:
this was the only occurrence of the lower-cased version of `uris` so it doesn't matter that much. We can uppercase it -- I just didn't want to have to add an exception to the dictionary for `uris`.

Likely we should just capitalize this

FWIW there is no 'official' distinction between `URI` and `URL`, but the practical distinction is:

> A URL is a type of URI that identifies a resource via a representation of its primary access mechanism (e.g., its network "location"), rather than by some other attributes it may have. 
> -- [W3C URIs, URLs and URNs](https://www.w3.org/TR/2001/NOTE-uri-clarification-20010921/)

Reply:
I would uppercase the exception and always use URL and URLs.

Reply:
I second @uranusjr .